.travis.yml

```diff
@@ -33,6 +33,10 @@ install:
       pip install -U "pip==10.0.1"
       PIP_ARGS="-c requirements/3.3/constraints.txt"
     fi
+  - |
+    if [[ $TRAVIS_PYTHON_VERSION == "3.4" ]]; then
+      pip install -U setuptools
```

Comment:
Why doesn't `pip` handle this for us? From what I can tell `pip` didn't drop support for Python 3.4 until 19.2, but PEP 517 support went in in 19.0.

That said, I'm OK with whatever works here, this is all basically written on sand.

Reply:
We need setuptools 52.0, let me fix pyproject.toml.

Reply:
Nope, setuptools drop support for 3.4 in 44.0.0.

As said... if we plan to drop 3.4, I'd just land this to make the CI pass and we can look at it if we really care 🤷 .